src/muya/lib/parser/marked/renderer.js

```diff
@@ -103,7 +103,7 @@ Renderer.prototype.listitem = function (text, checked) {
   }
 
   // task list
-  return '<li><input type="checkbox"' +
+  return '<li style="list-style-type: none;"><input type="checkbox"' +
```

Comment:
Is there a better way to separate the style from `li` tag,  maybe add a `class` name to `li` tag is better?